mod-circulation/src/main/resources/vega/mod-circulation/features/loans.feature

```diff
@@ -777,4 +777,34 @@ Feature: Loans tests
     Then status 200
     * def accounts = response.accounts;
     And match accounts[0].paymentStatus.name == 'Suspended claim returned'
-    And match accounts[1].paymentStatus.name == 'Suspended claim returned'
\ No newline at end of file
+    And match accounts[1].paymentStatus.name == 'Suspended claim returned'
+
+  Scenario: When an existing loan is checked in, update checkInServicePointId, returnDate
+    * def extItemBarcode = 'FAT-995IBC'
+    * def extUserBarcode = 'FAT-995UBC'
+    * def extServicePointId = call uuid1
+
+    # location and service point setup
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostLocation')
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostServicePoint') { servicePointId: #(extServicePointId) }
+
+    # post an item
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostInstance')
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostHoldings')
+    * def postItemResponse = call read('classpath:vega/mod-circulation/features/util/initData.feature@PostItem') { extItemBarcode: #(extItemBarcode) }
+    * def extItemId = postItemResponse.response.id
+
+    # post a group and user
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostGroup') { extUserGroupId: #(groupId) }
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostUser') { extUserBarcode: #(extUserBarcode) }
+
+    # checkOut the item
+    * def checkOutResponse = call read('classpath:vega/mod-circulation/features/util/initData.feature@PostCheckOut') { extCheckOutUserBarcode: #(extUserBarcode), extCheckOutItemBarcode: #(extItemBarcode) }
+    * def extLoanId = checkOutResponse.response.id
+
+    # checkIn the item and verify that checkInServicePointId and returnDate of the loan are updated
+    * def extCheckInDate = '2022-04-27T13:30:46.000Z'
```

Comment:
Could you please give a brief explanation of why we are using check-in date with hard code?

Reply:
It was just to check whether the date had been set or not as value of return date.
I fixed it to avoid future unpleasant errors